mobile/android/app/src/local/google-services.json

```diff
@@ -9,7 +9,7 @@
       "client_info": {
         "mobilesdk_app_id": "1:958970316277:android:c8a1d7a58212e3111e0cbc",
         "android_client_info": {
-          "package_name": "finance.get10101.app.demo"
+          "package_name": "finance.get10101.app.local"
```

Comment:
🤔 are you sure this would not break any of the other things that depend on having variant named "demo"? such as our regtest app? 

I guess in the worst case we can revert this simple change if it won't deploy the demo app.

Reply:
Unfortunately, I'm not sure, as I don't know how to test this. However, this only modifies the `google-services.json` for the `local` flavour (is flavour the right word?) and doesn't modify it for the `demo` flavour.

Reply:
I hinted that I don't mind testing it by trying it out; let's just be prepared for chance of reverting it if anything happens. (which should not be a problem with such a self-contained small commit) :-)

Reply:
Note: this would break push-notifications for local setup (provided it was working in the first place). FCM might throw an error on startup now.
We will need a special google-services.json config for local if we want to have push notification for `.local`.